guides/use-microprofile-amqp-connector-with-ssl-openshift.adoc

```diff
@@ -0,0 +1,208 @@
+/subsystem=elytron/key-store=keystore-ssl-test:add(credential-reference={clear-text=changeit}, path=<path-to>/client.ks, type=PKCS12)
+/subsystem=elytron/key-manager=keystore-ssl-test:add(key-store=keystore-ssl-test,credential-reference={clear-text=changeit})
+/subsystem=elytron/client-ssl-context=amqp-ssl-context:add(trust-manager=truststore-ssl-test,key-manager=keystore-ssl-test)
+run-batch
+----
+[NOTE]
+--
+Update `<path-to>` in above cli script to directory with generated truststore and keystore.
+--
+
+You can build the application by running:
+[source, bash]
+----
+mvn package
+----
+
+The application uses the WildFly Maven Plugin to provision an application server that is trimmed with only the required modules to run the application. It deploys reactive messaging app into it and package all as Bootable Jar which can be started like:
+[source, bash]
+----
+AMQ_HOST=<amq-broker-route> java -jar ./target/server-bootable.jar
+----
+
+You can figure out value for `AMQ_HOST` by checking `oc get routes` for `amq-broker-all-0-svc-rte` route.  Like:
+```
+$ oc get routes
+NAME                          HOST/PORT                                                                     PATH   SERVICES                  PORT       TERMINATION        WILDCARD
+amq-broker-all-0-svc-rte      amq-broker-all-0-svc-rte-mnovak.apps.eapqe-034-nvf2.eapqe.psi.redhat.com             amq-broker-all-0-svc      all-0      passthrough/None   None
+...
+```
+
+Once application starts you should see output like:
+[source, bash]
+----
+17:28:45,819 INFO  [stdout] (vert.x-eventloop-thread-0) ----> Calling generate!!!!
+17:28:45,820 INFO  [stdout] (vert.x-eventloop-thread-0) =====> Creating Next with 1
+17:28:45,821 INFO  [io.smallrye.reactive.messaging.amqp] (vert.x-eventloop-thread-0) SRMSG16203: AMQP Receiver listening address source
+17:28:45,921 INFO  [stdout] (pool-13-thread-1) ---> Sending 1
+17:28:45,926 INFO  [stdout] (pool-13-thread-1) ----> Calling generate!!!!
+17:28:45,927 INFO  [stdout] (pool-13-thread-1) =====> Creating Next with 2
+17:28:46,027 INFO  [stdout] (pool-13-thread-1) ---> Sending 2
+17:28:46,027 INFO  [stdout] (pool-13-thread-1) ----> Calling generate!!!!
+17:28:46,027 INFO  [stdout] (pool-13-thread-1) =====> Creating Next with 4
+17:28:46,046 INFO  [stdout] (vert.x-eventloop-thread-0) ---> Received 1
+17:28:46,128 INFO  [stdout] (pool-13-thread-1) ---> Sending 4
+17:28:46,128 INFO  [stdout] (pool-13-thread-1) ----> Calling generate!!!!
+17:28:46,128 INFO  [stdout] (pool-13-thread-1) =====> Creating Next with 8
+17:28:46,140 INFO  [stdout] (vert.x-eventloop-thread-0) ---> Received 2
+----
+
+== What's next?
+
+Wildfly MicroProfile Reactive Messaging provides multiple options to connect to different messaging brokers like Kafka. Read the configuration from https://docs.wildfly.org/31/Admin_Guide.html#MicroProfile_Reactive_Messaging_SmallRye[MicroProfile Reactive Messaging Subsystem Configuration]
```

Comment:
s/Wildfly/WildFly

Reply:
Fixed (also in other occurences), thanks.